extensions/wikia/Recirculation/js/experiments/placement.js

```diff
@@ -4,14 +4,30 @@ require([
 	'wikia.window',
 	'wikia.abTest',
 	'ext.wikia.recirculation.tracker',
+	'ext.wikia.recirculation.utils',
 	'ext.wikia.recirculation.views.incontent',
 	'ext.wikia.recirculation.views.rail',
 	'ext.wikia.recirculation.views.footer',
 	'ext.wikia.recirculation.helpers.contentLinks',
 	'ext.wikia.recirculation.helpers.fandom',
+	'ext.wikia.recirculation.helpers.googleMatch',
 	'ext.wikia.adEngine.taboolaHelper',
 	require.optional('videosmodule.controllers.rail')
-], function($, w, abTest, tracker, incontentView, railView, footerView, contentLinksHelper, fandomHelper, taboolaHelper, videosModule) {
+], function(
+	$,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has 13 parameters which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AExcessiveParameterList)